examples/source/1.components/amp-script/index.html

```diff
@@ -158,6 +173,36 @@
         fetchCurrentTime();
       </script>
     </div>
+
+    <!--
+      ## Using a WebSocket for a live blog
+      `amp-script` supports WebSockets. This example simulates a live blog.
+    -->
+    <div>
+      <amp-script layout="fixed-height" height="200" script="live-blog-script" class="sample" sandbox="allow-forms">
+        <button id="live-blog-start">Start live blog</button>
+        <div id="live-blog-area"></div>
+      </amp-script>
+  
+      <script id="live-blog-script" type="text/plain" target="amp-script">
+        const button = document.getElementById('live-blog-start');
+        const blogDiv = document.getElementById('live-blog-area');
+  
+        button.addEventListener("click", () => {
+          button.setAttribute('disabled', '');
+          button.textContent = 'Live blog begun';
+          const socket = new WebSocket('ws://localhost:8080/documentation/examples/api/socket/live-blog');
```

Comment:
This is not going to work once deployed. Use `<% host.platform %>` for the host.

Reply:
argh! I got distracted by all my trouble getting the route to work. Thanks for noticing this.

Reply:
Actually, do I have access to a `host` property that doesn't include the scheme? Otherwise I get an unwanted "http://"

Reply:
I didn't spot anything like that in `config.js` or the html files.

Is it possible to do something like this? (Though it's probably better to make url parts available to other html files via backend changes)

`<% hosts.platform.replace(/.+\/\//, "ws://") %>`

Reply:
This would be the right place to add a new variable `hosts.websocket`.

Reply:
Ok, did this. It also gave me a chance to learn more about how amp.dev's configuration is built and how that propagates into samples....